docs/ensembl-help/using-ensembl/getting-ensembl-data/tools-and-apis/graphql/getting-started.md

```diff
@@ -0,0 +1,276 @@
+---
+slug: accessing-graphql
+title: Accessing Ensembl GraphQL services
+description: How to access the GraphQL services provided by Ensembl
+related_articles:
+  - href: ./using-graphql.md
+tags:
+  - api
+  - tools
+status: draft
```

Comment:
No need for this field. Personally, I don't think the tags field is needed either; but status: draft is definitely not needed.

Reply:
I think I was just copying what was already there in other [pages](https://github.com/Ensembl/ensembl-website-help-and-docs/blob/7bf5f64b25873367f336487f7a04fd2f048d7a6d/docs/ensembl-help/getting-started/species.md?plain=1#L9C14-L9C14)

Should it be removed?

Reply:
Yes, they don't do anything.

I think Anne's idea for the `tags` field was that they could be used by content writers when they search the code base. This will hardly be helpful here.

Reply:
@azangru I think `tags` and `status` [date back to when we were discussing using CMS/(Prismic?)](https://genomes-ebi.slack.com/archives/C010X2BA56U/p1586862037003100)

I can think of three options:

1. Leave it as it is and change the `status` to `published`
2. Delete `status`
3. Delete both `tags` and `status`

I'm inclined to go with option 1 since there is no harm in leaving them there, they may come in handy in the future.
Or we can delete them (we should do the same for the other documents too)

What do you think @bethflint, @azangru?

Reply:
> I think tags and status [date back to when we were discussing using CMS/(Prismic?)](https://genomes-ebi.slack.com/archives/C010X2BA56U/p1586862037003100)

Strapi, I believe. But any CMSs we discussed were not based on markdown. Most likely, the status and the tags fields in the yaml appeared before the conventions for authoring the documents were solidified (some remnants of an early discussion have been preserved [here](https://github.com/Ensembl/ensembl-website-help-and-docs/blob/7bf5f64b25873367f336487f7a04fd2f048d7a6d/development_notes/meetings.md?plain=1#L1-L3))

Personally, I would go with 3, to follow the principle of the least surprise, because I don't like to have things in the code that look like instructions, but that that don't in fact do anything. One might argue that the word "draft" can be used as a keyword to search the codebase for draft articles; but my response to this would be that you can bet that people will forget to change the word `draft` to `published` before they hit merge; and since there is no automated mechanism for updating this keyword, and because these articles will still be visible in the UI despite the `draft` status, we will easily end up in a situation when a bunch of finished articles will continue claiming that they are drafts.